ironfish/src/node.ts

```diff
@@ -335,10 +335,12 @@ export class IronfishNode {
       this.rpc.stop(),
       this.telemetry.stop(),
       this.metrics.stop(),
-      this.workerPool.stop(),
       this.minedBlocksIndexer.stop(),
     ])
 
+    // Do after to avoid unhandled error from aborted jobs
+    await Promise.allSettled([this.workerPool.stop()])
```

Comment:
This is a bug because stopping the worker pool aborts jobs, which almost no job actually handles. By doing this before all the services are shut down, it cause the process to crash MOST times because of the unhandled JobAborted error. It was preventing clean shutdown from working correctly.